AntennaTracker/Tracker.h

```diff
@@ -154,6 +154,13 @@ class Tracker : public AP_Vehicle {
         bool need_altitude_calibration  : 1;// true if tracker altitude has not been determined (true after startup)
         bool scan_reverse_pitch         : 1;// controls direction of pitch movement in SCAN mode
         bool scan_reverse_yaw           : 1;// controls direction of yaw movement in SCAN mode
+
+        // stashed values for logging purposes
+        float pidP_target;
+        float pidP_actual;
+        float pidY_target;
+        float pidY_actual;
```

Comment:
These need to be rotated into body frame, as the error being passed into the PID is in BF.

May end up passing body-frame target/measured when updating the PID